adapters/Pangle/PangleAdapter/GADMediationAdapterPangle.h

```diff
@@ -15,5 +15,12 @@
 #import <Foundation/Foundation.h>
 #import <GoogleMobileAds/GoogleMobileAds.h>
 
+typedef NS_ENUM(NSInteger,GADPangleErrorCode) {
+    // Missing server parameters.
+    GADPangleErrorMissingServerParameters = 1,
+    // Missing server parameters.
+    GADPangleErrorInvalidRequest = 101,
```

Comment:
Would prefer a more granular error code, invalid request can be caused by many reasons.

Reply:
solved